src/promptflow/promptflow/_cli/_pf/_flow.py

```diff
@@ -467,14 +469,32 @@ def serve_flow_csharp(args, source):
         pass
 
 
+def _resolve_python_flow_additional_includes(source) -> Path:
+    # Resolve flow additional includes
+    from promptflow import load_flow
+
+    flow = load_flow(source)
+    with FlowOperations._resolve_additional_includes(flow.path) as resolved_flow_path:
+        if resolved_flow_path == flow.path:
+            return source
+        # Copy resolved flow to temp folder if additional includes exists
+        # Note: DO NOT use resolved flow path directly, as when inner logic raise exception,
```

Comment:
still not well get the reason here, which process occupied

Reply:
offline synced